src/library/scala/collection/convert/JavaCollectionWrappers.scala

```diff
@@ -434,6 +439,8 @@ private[collection] object JavaCollectionWrappers extends Serializable {
 
     override def get(k: K) = Option(underlying get k)
 
+    override def getOrElseUpdate(key: K, op: => V): V = underlying.computeIfAbsent(key, _ => op)
```

Comment:
do we even need these overrides? and if so, can we just use `super[JMapWrapperLike].{getOrElseUpdate,updateWith}`?

Reply:
Thanks NthPortal I will try to answer these questions.

Reply:
Just to confirm, this works
```
override def getOrElseUpdate(key: K, op: => V): V = super[AbstractJMapWrapper].getOrElseUpdate(key, op)
```
and compiles to invoke static. However, I think it may be too obscure: since the code duplication is modest, it's easier to avoid adding an extra jump.

Reply:
while I admit that `super[Type]` is obscure, I'm not certain that's a good reason to avoid it, but I don't feel particularly strongly about it

Reply:
(By "obscure", I intended to say "brittle inheritance".)